### ISSUE TITLE
CI: MariaDB 10.11 installatiion errors on Ubuntu 22.04

### ISSUE THREAD

**Issue by junaruga** [brianmario/mysql2#1311]
I noticed the errors on installing mariadb-server-10.11 in Ubuntu 22.04 cases.

https://github.com/brianmario/mysql2/actions/runs/4198869001/jobs/7283047753
```
 + apt install -y -o Dpkg::Options::=--force-confnew mariadb-server-10.11 libmariadb-dev

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
Package mariadb-server-10.11 is a virtual package provided by:
  mariadb-server 1:10.11.1+maria~ubu2204 [Not candidate version]

E: Package 'mariadb-server-10.11' has no installation candidate
Error: Process completed with exit code 100.
```

**Comment by sodabrew** [comment]
10.11 just went GA today in fact! https://mariadb.org/mariadb-10-11-2-ga-now-available/

I put it into the CI matrix because it was already marked as an LTS release series.

Possible that there was some movement in the deb repos as a result of the promotion?

**Comment by sodabrew** [comment]
Resolved by #1326
